package.json

```diff
@@ -131,7 +131,7 @@
     "eslint-plugin-node-dependencies": "^0.9.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-regexp": "^1.0.0",
-    "eslint-plugin-svelte": "^2.0.0",
+    "eslint-plugin-svelte": "^2.6.0",
```

Comment:
I don't think this change is necessary. Should we change updateInternalDependencies?

Reply:
Sure, if you don't want it.

Reply:
@ota-meshi Please check https://github.com/ota-meshi/eslint-plugin-svelte/pull/223/files#diff-ed5b41335ff968c5cfd8035d4b8f8c8c0538b6746182d522adb0312eb9137fc5R14

Actually, I think we should use `"eslint-plugin-svelte": "link:.",` here.

Reply:
I don't want to use link. I want to point it to the npm registry because I want to check if the release was successful.